runtime/autoload/provider/pythonx.vim

```diff
@@ -8,7 +8,7 @@ let s:loaded_pythonx_provider = 1
 function! provider#pythonx#Require(host) abort
   " Python host arguments
   let prog = provider#python3#Prog()
-  let args = [prog, '-c', 'import sys; sys.path = [p for p in sys.path if p != ""]; import neovim; neovim.start_host()']
+  let args = [prog, '-c', 'import sys; sys.path = [p for p in sys.path if p != ""]; import pynvim; neovim.start_host()']
```

Comment:
> Have you even checked if this actually works?

Obviously, This line would fail with NameError. Please run and test the code if your change works.

Reply:
Ouch, embarassing miss by me.

This probably shouldn't have been a PR, new to github PRs and didn't know another way to run the test suite, sorry. I tried to run the test suite locally but due to mismatching environment/missing dependencies some tests fail on master and some tests don't run. I created the PR to run tests in a tested environment.